python/ray/ml/config.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Dict
+
+
+# Right now, RunConfig is just an arbitrary dict that specifies tune.run
```

Comment:
Can we document the schema here?

Reply:
This depends on the Tuner implementation (cc @xwjiang2010). I left a TODO to make this into a dataclass (since RunConfig is not tunable), so when we make that change, documentation will be included there.

Reply:
It feels odd to expect a dataclass for the RunConfig and a dict for the ScalingConfig. IMO we should either converge to Dicts only (which is needed to tune single parameters), or to accept `Union[Dict[str, Any], RunConfig]` (and `Union[Dict[str, Any], Trainer.ScalingConfig]`).